eng/helixcorefxtests.proj

```diff
@@ -0,0 +1,111 @@
+<Project InitialTargets="BuildHelixWorkItems" Sdk="Microsoft.DotNet.Helix.Sdk">
```

Comment:
Why `BuildHelixWorkItems` is in `InitialTargets`?

Is it because Arcade doesn't have a target that be can specified in `BeforeTargets` attribute for `BuildHelixWorkItems`?

Reply:
I couldn't get BeforeTargets="Test" to work.  This is pretty how the corefx repo sets things up in their eng\sendtohelix.proj.